_rules/auto-update-text-efbfc7.md

```diff
@@ -13,32 +13,32 @@ accessibility_requirements: # Remove whatever is not applicable
 input_aspects:
   - DOM Tree
   - CSS Styling
-acknowledgements:
+acknowledgments:
   authors:
     - Carlos Duarte
 ---
 
 ## Applicability
 
-The rule applies to any [visible][] [HTML element][] in an [HTML document][] if:
+The rule applies to any [visible][] [HTML element][] if:
 
-- **(not alone)** it is not the only [content][] in the [HTML document][]; and
 - **(changed)** the `innerText` property of the [element][html element] changes; and
 - **(no child changed)** the [element][html element] does not have [children][child] whose `innerText` property also changes; and
+- **(not alone)** the [element][html element] has [sibling][] or [ancestor][] elements with non-empty `innerText` properties; and
```

Comment:
Wouldn't that mean that if the only other content is not text (image, video, …) there is no problem?

Hum, that would only make the rule be inapplicable "too often", so not a big deal anyway…

Reply:
To keep the rule unambiguous I'd leave it as is, risking some inapplicable cases, as you said.
However, re-reading this, I found that the `innerText` of the ancestors needs to be different from the `innerText` of the target element, otherwise something like the example below would be applicable because the outer `<div>` has inner text.
```html
<div>
  <div>auto-updating text</div>
</div>
```